linera-chain/src/inbox.rs

```diff
@@ -73,9 +73,9 @@ pub struct Cursor {
 
 #[derive(Error, Debug)]
 pub(crate) enum InboxError {
-    #[error(transparent)]
+    #[error("View error: {0}")]
```

Comment:
I'm not sure about this. Why is better?

Reply:
I am of course fine with having "transparent". But then we should have that for ALL cases of "ViewError".
I want consistency in the code.